adb_graphics/datahandler/grib.py

```diff
@@ -35,48 +35,33 @@ def _load(self):
         ''' Internal method that opens the grib file. Returns a grib message
         iterator. '''
 
-        return Nio.open_file(self.filename, format="grib2") # pylint: disable=c-extension-no-member
+        #return Nio.open_file(self.filename, format="grib2") # pylint: disable=c-extension-no-member
+        return xr.open_dataset(self.filename,
+                               engine='pynio',
+                               lock=False,
+                               backend_kwargs=dict(format="grib2"),
+                               )
```

Comment:
@Brian-Jamison The `grib.GribFile` class here in `adb_graphics/datahandler/grib.py`. It has been reduced to just opening the grib file and is no longer used as a base class for the other classes defined in this file. 

The only change that really happened to this part of the code is the way we load the data. Before we were using `Nio.open_file()` to get a `Nio` object. Now we use `xr.open_dataset()` with the `pynio` engine and load an XArray data structure. That change in object type was what necessitated a bunch of the changes for accessing the data inside. See the mods below that go from `filename` or`contents` to `ds`.